{"name": "test_2[5-6]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 6", "trace": "y1 = 5, y2 = 6\n\n    @pytest.mark.parametrize(\"y1,y2\",d2)\n    def test_2(y1,y2):\n        print(f\"y1的值是{y1}\")\n        print(f\"y2的值是{y2}\")\n>       assert y1 == y2\nE       assert 5 == 6\n\ntest_03.py:170: AssertionError"}, "attachments": [{"name": "stdout", "source": "edeb0a8c-be76-4dc8-a197-bed158a6d2b9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "y1", "value": "5"}, {"name": "y2", "value": "6"}], "start": 1565664406776, "stop": 1565664406777, "uuid": "fe9f87b2-a73c-47a7-ad65-66ee26b18eb2", "historyId": "9c496a2f052c6aebedeaed23ef168d11", "fullName": "test_03#test_2[5-6]", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_03"}, {"name": "subSuite"}, {"name": "host", "value": "sunzl"}, {"name": "thread", "value": "8752-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_03"}]}